aten/src/ATen/native/native_functions.yaml

```diff
@@ -13812,11 +13812,18 @@
   dispatch:
     CPU, CUDA: linalg_eig_out
 
+- func: _linalg_eigvals(Tensor self) -> Tensor
```

Comment:
Why adding this, rather than adding a dispatch to `linalg_eigvals`?

Reply:
When a derivative is needed, `linalg_eigvals` defers to `linalg_eig` because the eigenvectors are needed. For this to work, the operator needs to be `CompositeImplicitAutograd`.